### DISCUSSION TITLE
Routing consecutive operations

### DISCUSSION THREAD

**Discussion by ErneD** [frePPLe/frepple#600] [Q&A] [answered]
Hello everyone !

I have created a routing operation, very much like the one in the demo: painting an item. This routing has two steps:

1. Painting (which consumes materials and capacity)
2. Drying

These two operations needs to be executed one right after the other: the drying starts allways when the painting is done.
But sometimes, depending on the load, Frepple separates these two operatios and I get the drying operation starting like 1 week after the painting has finished !
Can anyone tell me how can I configure Frepple so the drying operation starts ALLWAYS right after the painting one has finished ?

Thank you !

Ernesto.

PS: I know I could remove the drying operation and set an offset in the operations materials for the painting to model the drying time  but I would like to have that drying as an operation and not as an invisible offset time.

**Comment by jdetaeye** [comment]
No special configuration is needed - sequence within a routing should always be respected.  This looks like a bug.

**Reply by ErneD** [reply]
Hi jdtaeye,
Thanks for your quick answer !
Ok, if I understand correctly the standard behaviour is : operations within a routing shouldn't be separated.
I am using a Frepple 8.0.0 Comunity docker version, perhaps this is already fixed in a later one ?
Great piece of software by th way !

**Reply by ErneD** [reply]
Just to make sure...
Sequence is respected: drying starts always after painting.
It's just that these two operations shouldn't be separated.

**Reply by jdetaeye** [reply, marked as answer]
Sorry, I misunderstood (and/or answered too quickly...).

If sequence is respected, it's not a bug.

There is no functionality that guarantees that routings steps are planned back-to-back without separation.   So, using the offset as you describe is your only option to achieve this.

**Reply by ErneD** [reply]
Ok. Thanks for your time !
